apps/web/cypress/e2e/multi-dwelling/multi-dwelling-9.9.9-walkthrough.cy.ts

```diff
@@ -30,6 +38,26 @@ describe("multi dwelling: 9.9.9 walkthrough", () => {
     ).should("be.hidden");
   });
 
+  it.only("validate p18 shows value from previous question p5", () => {
+    // This walkthrough should hit both P5 and P18
+    const walkthroughContainingP5P18 = getWalkthrough(walkthroughs, 1);
+    let p5AnswerValue = "";
+
+    for (const step of walkthroughContainingP5P18.steps) {
+      if (step.question === "P5") {
+        p5AnswerValue = step.answer;
+      }
+      if (step.question === "P18") {
+        cy.getByTestID(TESTID_QUESTION).contains(
+          `Does this dwelling unit have separate and direct access from each storey to a balcony on storey(s): ${p5AnswerValue}?`,
+        );
+        break;
```

Comment:
this lets us stop running the walkthrough at p18 so we don't have to spend time answering the rest of the questions that we don't care about